Source/Widgets/Timeline/Timeline.css

```diff
@@ -4,6 +4,7 @@
   bottom: 0;
   overflow: hidden;
   border: solid 1px #888;
+  user-select: none;
```

Comment:
It looks like anywhere else in the css where we apply this rule, we apply all the browser-specific rules as well. Let's do so here for consistency. 

```
  -moz-user-select: none;
  -webkit-user-select: none;
  -ms-user-select: none;
  user-select: none;
```

Reply:
Added that in. Thanks @ggetz